Magento_Customer/layout/default.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<page
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd"
+>
+    <body>
+        <referenceBlock
+            name="authentication-popup"
+            remove="true"
+        />
+    </body>
+</page>
```

Comment:
it's ok, more recaptcha issue, authentication-popup template is not loading anymore but there is still `Magento_Customer/js/model/authentication-popup.js` loading in header. Could you please check if we can disable this? what I've check quickly it's needed only on cart & checkout view. 
It should be also removed from common magepack config

Reply:
`'Magento_Customer/js/model/authentication-popup'` is used in `vendor/magento/module-checkout/view/frontend/web/js/sidebar.js`
`sidebar.js` is used in `vendor/magento/module-checkout/view/frontend/web/js/view/minicart.js`

We have minicart in all pages and that's why it's loaded even on homepage

So to remove it I have to override `minicart.js` file and remove `'sidebar'` from `define()`

Do you see better solution?

Reply:
ok, I think we should leave it for now. This optimisation require more work, without sidebar imported, minicart it's broken (Go to checkout button not working, I did a quick test). :(